clinicadl/maps/data_group/datagroup.py

```diff
@@ -0,0 +1,62 @@
+from abc import abstractmethod
+from pathlib import Path
+
+from clinicadl.data.datasets import CapsDataset
+from clinicadl.dictionary.suffixes import JSON, TSV
+from clinicadl.dictionary.words import (
+    DATA,
+    MAPS,
+    SPLIT,
+)
+from clinicadl.tsvtools.utils import df_to_tsv
+from clinicadl.utils.exceptions import ClinicaDLConfigurationError
+from clinicadl.utils.typing import PathType
+
+from ..base import Directory
+
+
+class BaseDataGroup(Directory):
+    @property
+    @abstractmethod
+    def name(self) -> str:
+        pass
+
+    @property
+    def data_tsv(self) -> Path:
+        return self.path / (DATA + TSV)
```

Comment:
```suggestion
        return (self.path / DATA).with_suffix(TSV)
```

Maybe a bit more explicit. WDYT?